tf-modules/distribution/outputs.tf

```diff
@@ -1,5 +1,5 @@
 output "s3_credentials_redirect_uri" {
-  value = var.deploy_s3_credentials_endpoint ? "${var.tea_internal_api_endpoint}redirect" : null
```

Comment:
Why did this use internal api endpoint before?

Reply:
Not sure.  But it doesn't work if I want to add port to the tea distribution url.
`tea_internal_api_endpoint` is the api gateway endpoint. 
`tea_external_api_endpoint` is the tea distribution endpoint which can be set with terraform variable `tea_distribution_url`.
e.g.
tea_internal_api_endpoint = https://tea-endpoint.execute-api.us-east-1.amazonaws.com/DEV/
tea_external_api_endpoint = https://tea-endpoint.execute-api.us-east-1.amazonaws.com:7000/DEV/ or cloudfront url